guides/source/command_line.md

```diff
@@ -29,10 +29,10 @@ There are a few commands that are absolutely critical to your everyday usage of
 * `bin/rails dbconsole`
 * `rails new app_name`
 
-You can get a list of rails commands available to you, which will often depend on your current directory, by typing `rails --help`. Each command has a description, and should help you find the thing you need.
+You can get a list of rails commands available to you, which will often depend on your current directory, by typing `bin/rails --help`. Each command has a description, and should help you find the thing you need.
```

Comment:
> which will often depend on your current directory

I think this is about how running `rails --help` outside of an application directory prints the help for `rails new`, whereas inside an application directory it prints the output shows below. Now that we're talking about `bin/rails` specifically, this distinction no longer makes sense.